examples/grid2grid/evaUtilities.py

```diff
@@ -0,0 +1,132 @@
+import matplotlib.pyplot as plt
+from config.config_manager import load_use_case_config
+import numpy as np
+from mpl_toolkits.mplot3d import proj3d
+from matplotlib.patches import Rectangle, Circle, FancyBboxPatch, FancyArrowPatch
+import mpl_toolkits.mplot3d.art3d as art3d
+import mpl_toolkits.mplot3d as mp3d
+import matplotlib.patches as mpatches
+
+
+class Arrow3D(FancyArrowPatch):
+    def __init__(self, xs, ys, zs, *args, **kwargs):
+        FancyArrowPatch.__init__(self, (0,0), (0,0), *args, **kwargs)
+        self._verts3d = xs, ys, zs
+
+    def draw(self, renderer):
+        xs3d, ys3d, zs3d = self._verts3d
+        xs, ys, zs = proj3d.proj_transform(xs3d, ys3d, zs3d, renderer.M)
+        self.set_positions((xs[0], ys[0]), (xs[1], ys[1]))
+        FancyArrowPatch.draw(self, renderer)
+
+
+def wrap_to_pi(angle):
+    # Wrap to 360
+    ang_2pi = angle - (angle // (2 * np.pi)) * 2*np.pi
+    if ang_2pi > np.pi or ang_2pi < - np.pi:
+        # Wrap to 180
+        ang_pi = ang_2pi-np.sign(ang_2pi)*2*np.pi
+        return ang_pi
+
+
+def quaternion_multiply(quaternion1, quaternion0):
+    w0, x0, y0, z0 = quaternion0
+    w1, x1, y1, z1 = quaternion1
+    return np.array([-x1 * x0 - y1 * y0 - z1 * z0 + w1 * w0,
+                     x1 * w0 + y1 * z0 - z1 * y0 + w1 * x0,
+                     -x1 * z0 + y1 * w0 + z1 * x0 + w1 * y0,
+                     x1 * y0 - y1 * x0 + z1 * w0 + w1 * z0], dtype=np.float64)
+
+
+def solve_ik(_eva, guess, theta, xyz_absolute):

```

Comment:
inconsistent input variable names, _eva -> eva

Reply:
changed this